### PR TITLE
F-Droid page resources

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
Introduces the necessary resources for the APK to display description and pictures on F-Droid repos.

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: mobile/fastlane/metadata/en-US/full_description.txt
Comment (IzzySoft) [inline mobile/fastlane/metadata/en-US/full_description.txt:1]:
```html
<p>RoboSats is a simple and private app to exchange bitcoin for national currencies. Robosats simplifies the P2P user experience and uses lightning hold invoices to minimize custody and trust requirements. The deterministically generated robot avatars help users stick to best privacy practices.</p>
<p><br><b>Features:</b></p><ul>
<li>Privacy focused: your robot avatar is deterministically generated, no need for registration.</li>
<li>…</li>
</ul>
<p>You can join other cool Robots and get community support at <a href="https://t.me/robosats">our Telegram group</a>.</p>

**Review by IzzySoft — CHANGES_REQUESTED** [review]
all else: LGTM. Items like featureGraphic and per-release changelogs could be added later, if wanted.

File: mobile/fastlane/metadata/en-US/full_description.txt
Comment (KoalaSat) [reply mobile/fastlane/metadata/en-US/full_description.txt:1]: thanks!

**Comment by Reckless-Satoshi** [comment]
Thank you @KoalaSat Looks good to merge. Please post an invoice for 100K Sats for a devfund reward.

**Comment by KoalaSat** [comment]
> Thank you @KoalaSat Looks good to merge. Please post an invoice for 100K Sats for a devfund reward.

lnbc1m1pnyz3gmpp5rm9zxrp7q46fkqv4znumzz06ty49a6uspdgde5qe5nzlshrpe2gqdqqcqzzsxqys73rsp5qqc7wlnc6nkrca769afz4aseesl4usjypudu4gd7mp9trx78x6cq9qyyssq5ljdl3qt8npgjd8whlqt7ahclw6frlznc9cx03fzsck6xlm0qhd8l4mnq06n96xu99n5wvzfa8sg8ru5afaj6lvx0nn2q8l4mtjy5wsq3rqxx3

**Comment by Reckless-Satoshi** [comment]
> lnbc1m1pnyz3gmpp5rm9zxrp7q46fkqv4znumzz06ty49a6uspdgde5qe5nzlshrpe2gqdqqcqzzsxqys73rsp5qqc7wlnc6nkrca769afz4aseesl4usjypudu4gd7mp9trx78x6cq9qyyssq5ljdl3qt8npgjd8whlqt7ahclw6frlznc9cx03fzsck6xlm0qhd8l4mnq06n96xu99n5wvzfa8sg8ru5afaj6lvx0nn2q8l4mtjy5wsq3rqxx3

cca6dc5218dd61dbbe8acd8c7cade558c331c60f6e0faf37d952083e21875088

**Pull request merged**